pipelines/build/common/config_regeneration.groovy

```diff
@@ -60,6 +62,7 @@ class Regeneration implements Serializable {
         this.javaVersion = javaVersion
         this.buildConfigurations = buildConfigurations
         this.targetConfigurations = targetConfigurations
+        this.activeNodeTimeout = activeNodeTimeout
```

Comment:
Can be removed if we're removing if we're not specifying a node timeout in the regen jobs